x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/components/missing_privileges/index.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useMemo } from 'react';
+import { useKibana } from '../../../../common/lib/kibana';
+import {
+  MissingPrivilegesDescription,
+  MissingPrivilegesTooltip,
+} from '../../../../common/missing_privileges';
+import type { MissingCapability } from '../../service/capabilities';
+
+export const WithMissingPrivilegesTooltip = <
+  P extends { missingCapabilities: MissingCapability[] }
+>(
+  Component: React.ComponentType<P>
+) => {
+  const WrappedComponent = memo(function WrappedComponent(props: Omit<P, 'missingCapabilities'>) {
+    const {
+      services: { siemMigrations },
+    } = useKibana();
```

Comment:
super NIT: we can do this in one line
```suggestion
    const { siemMigrations } = useKibana().services;
```